Core/include/Acts/Material/Material.hpp

```diff
@@ -83,9 +83,9 @@ class Material {
   Material& operator=(const Material& mat) = default;
 
   /// Check if the material is valid, i.e. it is not vacuum.
-  constexpr operator bool() const { return 0.0f < m_ar; }
+  bool valid() const { return 0.0f < m_ar; }
```

Comment:
here I used `isValid` https://github.com/acts-project/acts/blob/727d4b12817f612b8a1c557bb9e58969654fbdbf/Core/include/Acts/Utilities/Intersection.hpp#L70 maybe we can make this sound